src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtil.java

```diff
@@ -589,4 +589,16 @@ private static int lastIndexOfNonWhitespace(String line) {
         }
         return length;
     }
+
+    /**
+     * Checks if a token type is a literal true or false.
+     *
+     * @param tokenType the TokenType
+     * @return true iff tokenType is LITERAL_TRUE or LITERAL_FALSE
```

Comment:
`iff`` -> if, please do me a favor and correct this typo as a part of extraction commit